src/backend/SparseArray.cpp

```diff
@@ -140,6 +146,11 @@ SparseArray<T> createArrayDataSparseArray(
 }
 
 template<typename T>
+SparseArray<T> copySparseArray(const SparseArray<T>& other, bool copy) {
+  return SparseArray<T>(other, copy);
+}
+
```

Comment:
Why do you need a `copy` parameter ? I don't see it being set to `false` anywhere.